edx_solutions_api_integration/management/commands/convert_ooyala_embeds.py

```diff
@@ -0,0 +1,74 @@
+import logging
+from optparse import make_option
+import datetime
+
+from pytz import UTC
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from edx_solutions_api_integration.tasks import convert_ooyala_embeds
+
+logger = logging.getLogger(__name__)  # pylint: disable=locally-disabled, invalid-name
+
+
+class Command(BaseCommand):
+    """
+    Command to update Ooyala Xblock Content IDs to corresponding Brightcove IDs
+    """
+    help = 'Convert Ooyala IDs to corresponding Brightcove IDs'
+    batch_size = 100
+    option_list = BaseCommand.option_list + (
+        make_option(
+            "--user-id",
+            dest="user_id",
+            help="Staff User ID",
+        ),
+        make_option(
+            "--course-id",
+            dest="course_id",
+            help="Single Course ID to process Ooyala instances in",
+        ),
+        make_option(
```

Comment:
i think revert option is not available for this command, please remove it.

Reply:
removed